src/org/javarosa/core/model/FormDef.java

```diff
@@ -1525,6 +1528,12 @@ private void attachControlsToInstanceData(TreeElement node) {
         }
     }
 
+    /**
+     * Returns the given question definition related to a form node reference.
```

Comment:
Using the word “given” here seems wrong. “given” suggests something given by the caller in method arguments.

Reply:
Agree w/ that. Let me edit it.

Reply:
Corrected this one too. Marking resolved.